.gitattributes

```diff
@@ -0,0 +1,13 @@
+* text=auto
+
+/docs export-ignore
+/examples export-ignore
+/test export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml export-ignore
+/travis.phpunit.xml.dist export-ignore
+/SECURITY.md export-ignore
+/changelog.md export-ignore
+/README.md export-ignore
```

Comment:
Yeah, I agree with @Synchro. These files just take up space in production. The only file important to keep is LICENSE.